vm_manager/helpers/pacemaker.py

```diff
@@ -23,6 +23,9 @@ class Pacemaker:
     Helper class to manipulate Pacemaker.
     """
 
+    _crmprog = ""
+    _crmresourceprog = ""
```

Comment:
For me the simplest solution to find crm is to let Linux finding crm in the PATH by removing the /usr/bin/ part.

Reply:
done